primeqa/ir/scripts/xortydi/convert_colbert_results_to_xor.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import pandas as pd
+import json
+
+logger=logging.getLogger(__name__)
+logging.basicConfig(level = logging.INFO)
+
+
+def handle_args():
+    usage='usage'	
+    parser=argparse.ArgumentParser(usage)
+    parser.add_argument('--corpus','-c', required=True)
+    parser.add_argument('--queries','-q', required=True)
+    parser.add_argument('--pred', '-p', required=True)
+    parser.add_argument('--out','-o', required=True)  # prefix
+    parser.add_argument('--topn', '-n', type=int, default=50)  # restrict output list of contexts to top n
+    parser.add_argument('--positional_qids_in_predictions', '-P', action="store_true", default=False) # the predictions file has 0, 1, 2 ..., as opposed to actual query IDs such as -"1018987027025691298"
```

Comment:
Do we need --positional_qids_in_predictions
My vote would be to remove it.  Generally datasets have an question id which should be used.

Reply:
We do have datasets (typically created as subsets/combinations of other sets) where the query file (used in search) has positional query IDs, while the ground truth file has the IDs from the original source, so I'd prefer to keep the option in place (and it defaults to False, so there is no burden on using files with "long" query IDs).